### PR TITLE
docs: max-lines for code editor

### PR DESCRIPTION
**Pull request by hasnain2808**
docs for https://github.com/frappe/frappe/pull/13966

### CODE REVIEW

File: frappe_docs/www/docs/user/en/api/controls.md [hunk 1]
```diff
@@ -76,7 +76,11 @@ Here are the `df` properties for most of frappe control types.
 	// for syntax highlighting
 	options: 'Javascript' // JS, HTML, CSS, Markdown, SCSS, JSON,
 	// wrap code
-	wrap: true
+	wrap: true,
+	// max-height of the editor would depend on the number of lines of code if this is set
+	max_lines: 10
```
Comment (gavindsouza) [inline frappe_docs/www/docs/user/en/api/controls.md:81]:
```suggestion
	max_lines: 10,
```

File: frappe_docs/www/docs/user/en/api/controls.md [hunk 2]
```diff
@@ -76,7 +76,11 @@ Here are the `df` properties for most of frappe control types.
 	// for syntax highlighting
 	options: 'Javascript' // JS, HTML, CSS, Markdown, SCSS, JSON,
 	// wrap code
-	wrap: true
+	wrap: true,
+	// max-height of the editor would depend on the number of lines of code if this is set
```
Comment (gavindsouza) [inline frappe_docs/www/docs/user/en/api/controls.md:80]: This and the next comment are unclear. if I set `max_lines` to 5, and there's 3 lines populated, what would be the `max-height`?

**Review by gavindsouza — CHANGES_REQUESTED** [review]

File: frappe_docs/www/docs/user/en/api/controls.md (hunk 2 above)
Comment (hasnain2808) [reply frappe_docs/www/docs/user/en/api/controls.md:80]: It is max lines, so unless you have lines more than that the height would be equivalent to the number of lines.
If you have also set min-lines, the height would always stay between the min-lines and max lines
These are boounds

Comment (gavindsouza) [reply frappe_docs/www/docs/user/en/api/controls.md:80]: Documentation is supposed to be simple. If I didn't understand your comment here, how do you expect another read to understand it? The comment is too wordy, and so is your reply; "of the editor would depend on the number of lines of code if this is set" can be condensed a lot IMO.

```suggestion
	// changing `max_lines` will set the max-height of the editor
```

Maybe ^ or something that directly states what is happening. 

In other terms, make docs DRY lol
